Makefile

```diff
@@ -336,39 +336,43 @@ ci-coverage-report: $(SUMMARY_COVER_PROFILE) coverage-report
 	go install github.com/mattn/goveralls@v0.0.7
 	@goveralls -coverprofile=$(SUMMARY_COVER_PROFILE) -service=buildkite || (printf $(RED) "Generating report for Coveralls (goveralls) failed."; exit 1)
 
+# optional args to make it possible to create multiple keyspaces:
+# make install-schema TEMPORAL_DB=temporal2 VISIBILITY_DB=temporal_visibility2
+TEMPORAL_DB?=temporal
+VISIBILITY_DB?=temporal_visibility
```

Comment:
I don't think that `make` has `?=` operator.

Can you also move these on top? Line 73 sounds like a good spot (after `PERSISTENCE_DRIVER`).

Reply:
It seems it has it: https://www.gnu.org/software/make/manual/html_node/Setting.html
Can you change all `ifndef` blocks in the `Makefile` to `?=` ?